functions/nvm.fish

```diff
@@ -196,8 +196,8 @@ function _nvm_use
         end
     end
 
-    if set -l root (_nvm_find_up (pwd) $nvm_file)
-        echo $argv[1] >$root/$nvm_file
+    if set -l root (_nvm_find_up (pwd) $nvm_file); and test $ver != (cat $root/$nvm_file)
+        echo $ver >$root/$nvm_file
```

Comment:
Thinking about this a bit more, it may be best to leave this as `argv[1]`, since the user may have actually intended to have a partial version specifier or alias in `.nvmrc`.  Does that seem right?

Reply:
yes!